<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admindashboard_students</class>
 <widget class="QWidget" name="admindashboard_students">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: #111;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;;
    background-color: transparent;
    padding-bottom: 5px;
   </string>
       </property>
       <property name="text">
        <string>Search by Student_Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="searchbutton">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchstudentid">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 2px solid #a0eec0;
    border-radius: 20px;
    padding: 8px 15px;
    font-size: 14px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #ff8591;
    background-color: #f9f9f9;
}

QLineEdit::placeholder {
    color: #95a5a6;
    font-style: italic;
}</string>
       </property>
       <property name="placeholderText">
        <string>Eg: 101</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: #111;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;;
    background-color: transparent;
    padding-bottom: 5px;
   </string>
       </property>
       <property name="text">
        <string>FIlter By Department,Class and Section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="departmentcombobox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="placeholderText">
        <string>Select Department</string>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Select Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Science</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Commerce</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classcombobox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="placeholderText">
        <string>Select Class</string>
       </property>
       <item>
        <property name="text">
         <string>Select Class</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sectioncombobox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="placeholderText">
        <string>Select Section</string>
       </property>
       <item>
        <property name="text">
         <string>Select Section</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="styleSheet">
         <string notr="true">QTableWidget {
    background-color: #ffffff;
    alternate-background-color: #f2f2f2;
    selection-background-color: #a0eec0;
    selection-color: #000000;
    border: 1px solid #dcdcdc;
    gridline-color: #dcdcdc;
}

QHeaderView::section {
    background-color: #f8f9fa;
    padding: 4px;
    border: 1px solid #dcdcdc;
    font-weight: bold;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border color : black;
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">color: rgb(152, 176, 255);
background-color: rgb(228, 255, 49);
font: 11pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Add a student</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="createyouraccount">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>Fill the form correctly</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sdform_name">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Enter  full name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sdform_email">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Enter student email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
           </property>
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="student_id">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Enter Student Id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Department</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sdform_dept">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select department</string>
           </property>
           <item>
            <property name="text">
             <string>Science</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Commerce</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Class</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sdform_class">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select class</string>
           </property>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Section</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sdform_section">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select section</string>
           </property>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSave">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(134, 160, 255);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletebutton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 53, 46);
color: rgb(0, 0, 0);
font: 600 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Delete Student</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
